.github/ISSUE_TEMPLATE/bug-issue-template.md

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
I believe we should use the new [**GitHub issue forms**](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms). They are way better and much easier to use, IMO.
Should be pretty easy to port this one. If you need any help, let us know here on our [**Discord server**](https://the-algorithms.com/discord/). 🙂

Reply:
@Panquesito7 that is a good point. I've converted the bug issue template to `yaml` format and added a template for the feature request.